ownCloud/Release Notes/ReleaseNotesHostViewController.swift

```diff
@@ -0,0 +1,211 @@
+	private let padding : CGFloat = 20.0
+	private let smallPadding : CGFloat = 10.0
+	private let buttonHeight : CGFloat = 44.0
+	private let headerHeight : CGFloat = 60.0
+
+	// MARK: - Instance Variables
+	var titleLabel = UILabel()
+	var proceedButton = UIButton(type: .roundedRect)
+	var footerLabel = UILabel()
+
+	override func viewDidLoad() {
+		super.viewDidLoad()
+
+		Theme.shared.register(client: self)
+
+		VendorServices.setUserPreferenceValue(NSString(utf8String: VendorServices.shared.appVersion), forClassSettingsKey: .lastSeenReleaseNotesVersion)
+
+		let headerView = UIView()
+		headerView.backgroundColor = .clear
+		headerView.translatesAutoresizingMaskIntoConstraints = false
+		view.addSubview(headerView)
+		NSLayoutConstraint.activate([
+			headerView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+			headerView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+			headerView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+			headerView.heightAnchor.constraint(equalToConstant: headerHeight)
+		])
+
+		titleLabel.translatesAutoresizingMaskIntoConstraints = false
+		titleLabel.setContentHuggingPriority(UILayoutPriority.defaultLow, for: NSLayoutConstraint.Axis.horizontal)
+
+		titleLabel.text = "New in ownCloud".localized
+		titleLabel.textAlignment = .center
+		titleLabel.numberOfLines = 0
+		titleLabel.font = UIFont.systemFont(ofSize: UIFont.systemFontSize * 1.5, weight: .bold)
+		headerView.addSubview(titleLabel)
+
+		NSLayoutConstraint.activate([
+			titleLabel.leftAnchor.constraint(greaterThanOrEqualTo: headerView.safeAreaLayoutGuide.leftAnchor, constant: padding),
+			titleLabel.rightAnchor.constraint(lessThanOrEqualTo: headerView.safeAreaLayoutGuide.rightAnchor, constant: padding * -1),
+			titleLabel.centerXAnchor.constraint(equalTo: headerView.safeAreaLayoutGuide.centerXAnchor),
+
+			titleLabel.topAnchor.constraint(equalTo: headerView.safeAreaLayoutGuide.topAnchor, constant: padding)
+		])
+
+		let releaseNotesController = ReleaseNotesTableViewController(style: .plain)
+		if let containerView = releaseNotesController.view {
+			containerView.backgroundColor = .clear
+			containerView.translatesAutoresizingMaskIntoConstraints = false
+			view.addSubview(containerView)
+
+			let bottomView = UIView()
+			bottomView.backgroundColor = .clear
+			bottomView.translatesAutoresizingMaskIntoConstraints = false
+			view.addSubview(bottomView)
+			NSLayoutConstraint.activate([
+				bottomView.leadingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.leadingAnchor),
+				bottomView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor),
+				bottomView.topAnchor.constraint(equalTo: containerView.bottomAnchor),
+				bottomView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor)
+			])
+
+			proceedButton.setTitle("Proceed".localized, for: .normal)
+			proceedButton.layer.cornerRadius = cornerRadius
```

Comment:
@hosy I noticed that we use different corner radius values for buttons, sometimes 8.0 and sometimes 10.0. Could we unify it and e.g. provide some category on `UIButton` with some factory method or similar?

Reply:
now using ThemeButton